File: app/src/main/java/com/kota/TelnetUI/TextView/TelnetTextViewUltraLarge.java
package com.kota.TelnetUI.TextView;

import android.content.Context;
import android.util.AttributeSet;

public class TelnetTextViewUltraLarge extends TelnetTextView {
    public TelnetTextViewUltraLarge(Context context, AttributeSet attrs, int defStyle) {
        super(context, attrs, defStyle);
        reloadTextSize();
    }

    public TelnetTextViewUltraLarge(Context context, AttributeSet attrs) {
        super(context, attrs);
        reloadTextSize();
    }

    public TelnetTextViewUltraLarge(Context context) {
        super(context);
        reloadTextSize();
    }

    /* access modifiers changed from: protected */
    public void reloadTextSize() {
        setTextModelSize(2);
    }
}

File: app/src/main/java/com/kota/telnetUI/textView/TelnetTextViewUltraLarge.kt
package com.kota.telnetUI.textView

import android.content.Context
import android.util.AttributeSet

class TelnetTextViewUltraLarge : TelnetTextView {
    constructor(context: Context?, attrs: AttributeSet?, defStyle: Int) : super(
        context!!,
        attrs,
        defStyle
    ) {
        reloadTextSize()
    }

    constructor(context: Context?, attrs: AttributeSet?) : super(context!!, attrs) {
        reloadTextSize()
    }

    constructor(context: Context?) : super(context!!) {
        reloadTextSize()
    }

    public override fun reloadTextSize() {
        setTextModelSize(TEXT_VIEW_SIZE_ULTRA_LARGE)
    }
}
